setup.py

```diff
@@ -51,7 +51,7 @@
         packages=find_packages(),
         install_requires=[
             "django-contrib-comments",
-            "django >= 1.11, < 2.1",
+            "django >= 2.2, < 3.0",
             "filebrowser_safe >= 0.5.0",
```

Comment:
in my case I needed to upgrade filebrowser_safe too

Reply:
filebrowser_safe has its now has its own PR with 2.2 compatible changes